src/System.CommandLine.Tests/Invocation/CancelOnProcessTerminationTests.cs

```diff
@@ -91,6 +91,170 @@ public async Task CancelOnProcessTermination_cancels_on_process_termination(int
             process.ExitCode.Should().Be(CancelledExitCode);
         }
 
+        [LinuxOnlyTheory]
+        [InlineData(SIGINT)]
+        [InlineData(SIGTERM)]
+        public async Task CancelOnProcessTermination_null_timeout_on_cancel_processing(int signo)
```

Comment:
Is this test redundant with the pre-existing test above?

Reply:
It is redundand with the current `CancelOnProcessTermination` implementation. But it guards the possibility of contract/behavior change on explict passing of `null` as `timeout`.
I can remove it if you feel it's superfluous.